src/operator/nn/dropout-inl.h

```diff
@@ -92,7 +92,7 @@ class DropoutOp {
       const int my_amount = std::min(my_offset + avg_amount, n) - my_offset;
       if (my_amount > 0) {
         VSLStreamStatePtr stream;
-        vslNewStream(&stream, VSL_BRNG_MCG31, seed + my_offset);
+        vslNewStream(&stream, VSL_BRNG_MCG31, seed);
```

Comment:
@pengzhao-intel vslNewStream was passed (seed + my_offset) which is variable for different threads as my_offset depends on thread number. I have removed my_offset, so whatever seed value is generated, it's used to generate stream for each thread.

Reply:
@apeforest By removing my_offset from vslNewStream and making the seed same for all threads, we are ensuring that the same stream is generated for each thread.
For each thread, my_offset acts an index for vslSkipAheadStream and viRngBernoulli. It tells vslSkipAheadStream which section of stream to use and viRngBernoulli which section of vector to populate.